Modules/_datetimemodule.c

```diff
@@ -5877,15 +5972,15 @@ datetime_timestamp(PyDateTime_DateTime *self, PyObject *Py_UNUSED(ignored))
 }
 
 static PyObject *
-datetime_getdate(PyDateTime_DateTime *self, PyObject *Py_UNUSED(ignored))
+datetime_getdate(PyDateTime_DateTime *self)
 {
     return new_date(GET_YEAR(self),
                     GET_MONTH(self),
                     GET_DAY(self));
 }
 
 static PyObject *
-datetime_gettime(PyDateTime_DateTime *self, PyObject *Py_UNUSED(ignored))
```

Comment:
I don't know why this `PyObject *Py_UNUSED(ignored)` is here, so per [Chesterton's Fence](https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence), I'm not comfortable removing it. Any insight as to why it is here and what the consequences will be in removing it?

Possibly it will be a breaking change in the C ABI?

Reply:
According to `git blame`, [this is to silence `gcc8`](https://github.com/python/cpython/commit/55edd0c185ad2d895b5d73e47d67049bc156b654). Is this still relevant @siddhesh @serhiy-storchaka?

Reply:
Yes, `datetime_gettime` should have two arguments, the second is ignored.